pkg/fetcher/fetcher.go

```diff
@@ -414,6 +429,58 @@ func (u *URLFetcher) IsNonretryableClientError() bool {
 		s != http.StatusLocked && s != http.StatusTooManyRequests
 }
 
+// Build error message for unexpected http code (nonretryable client errors and all other errors)
```

Comment:
Godoc comments for functions should start with the function name. See https://golang.org/doc/effective_go.html#commentary.